develop.sh

```diff
@@ -0,0 +1,78 @@
+#! /usr/bin/env bash
+set -euo pipefail
+
+expected_java_version="1.8"
+expected_maven_version="3"
+test_maven_version="3.6.3"
+
+jdk_help() {
+  echo "  Instructions for macOS:"
+  echo "    brew install openjdk@8"
+  echo "    export JAVA_HOME=\"\$(brew --prefix)/opt/openjdk@8/libexec/openjdk.jdk/Contents/Home\""
+}
+
+failure() {
+  echo
+  echo "Failure... See above for help setting things up."
+  exit 1
+}
+
+if [[ -z "${JAVA_HOME-}" ]]; then
+  echo
+  echo "JAVA_HOME pointing to JDK 1.8 is needed."
```

Comment:
I've been able to use JDK 11 all this time. The source configuration (https://github.com/snyk/snyk-maven-plugin/blob/master/pom.xml#L205) makes it work (forces it to be Java 8 compatible).